Source/SwiftLintFramework/Rules/Idiomatic/TypeNameRule.swift

```diff
@@ -92,3 +94,13 @@ public struct TypeNameRule: ASTRule, ConfigurationProviderRule {
         return []
     }
 }
+
+fileprivate extension String {
```

Comment:
nitpick: this can be a `private extension`

Reply:
Thank you.  TIL the above.  That is unexpected behaviour to me.  I thought `private extension` would mean that only `String` would see `nameStrippingTrailingSwiftUIPreviewProvider`